src/Modules/Data/Commands.Data/GetPowerBIDataset.cs

```diff
@@ -53,6 +53,16 @@ public class GetPowerBIDataset : PowerBIClientCmdlet, IUserScope, IUserFilter, I
         [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
         public int? First { get; set; }
 
+        [Alias("Expand")]
+        [Parameter(Mandatory = false, ParameterSetName = IdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ListParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectIdParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectNameParameterSetName)]
+        [Parameter(Mandatory = false, ParameterSetName = ObjectListParameterSetName)]
+        [ValidateSet("actualStorage")]
```

Comment:
Make capital and you might need to do empty string too, unless validateset only runs when something is specified

Reply:
ValidateSet only runs when specified, ignore that. Just make capital

Reply:
1) and lowercase it for request? i mean -- the request needs lcfirst
2)
```
> (Get-PowerBIDataset -Id 9139b36f-e02c-46dd-bfdd-224f757a6a28 -Include actualStorage -Scope Organization).Id

Guid
----
9139b36f-e02c-46dd-bfdd-224f757a6a28


> (Get-PowerBIDataset -Id 9139b36f-e02c-46dd-bfdd-224f757a6a28 -Scope Organization).Id

Guid
----
9139b36f-e02c-46dd-bfdd-224f757a6a28

```

looks ok